packages/odyssey-storybook/src/components/odyssey-mui/AdditionalContextIcon/AdditionalContextIcon.stories.tsx

```diff
@@ -0,0 +1,99 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Meta, StoryObj } from "@storybook/react";
+import {
+  AdditionalContextIcon,
+  AdditionalContextIconProps,
+} from "@okta/odyssey-react-mui";
+import { MuiThemeDecorator } from "../../../../.storybook/components";
+import { userEvent, within } from "@storybook/testing-library";
+import { axeRun } from "../../../axe-util";
+import type { PlaywrightProps } from "../storybookTypes";
+
+const storybookMeta: Meta<AdditionalContextIconProps> = {
+  title: "MUI Components/AdditionalContextIcon",
+  component: AdditionalContextIcon,
+  argTypes: {
+    text: {
+      control: "text",
+      description: "The text to display in the tooltip",
+      table: {
+        type: {
+          summary: "string",
+        },
+      },
+      type: {
+        required: true,
+        name: "string",
+      },
+    },
+    placement: {
+      options: ["top", "right", "bottom", "left"],
+      control: { type: "radio" },
+      description: "The placement of the tooltip",
+      table: {
+        type: {
+          summary: "top | right | bottom | left",
+        },
+        defaultValue: {
+          summary: "right",
+        },
+      },
+    },
+  },
+  args: {
+    testId: "tooltip",
+  },
+  decorators: [MuiThemeDecorator],
+  tags: ["autodocs"],
+};
+
+export default storybookMeta;
+
+const showTooltip =
+  ({ canvasElement, step }: PlaywrightProps<AdditionalContextIconProps>) =>
+  async (actionName: string) => {
+    await step("show the tooltip on hover", async () => {
+      const canvas = within(canvasElement);
+      const icon = canvas.getByTitle(
+        "Use the `i` icon to give details that might not be critical but are 'good to know'"
+      );
+      userEvent.hover(icon);
+      await axeRun(actionName);
+      userEvent.unhover(icon);
+    });
+  };
+
+const Template: StoryObj<AdditionalContextIconProps> = {
```

Comment:
I believe the generic typing for al story objects should be `StoryObj<typeof AdditionalContextIcon>`. I recall the typings like the one you are currently using to type this generic were to get around some storybook bug, but the suggestion here should work without the need for `AdditionalContextIconProps` here and other places.